app/assets/stylesheets/application.css

```diff
@@ -237,3 +237,8 @@ section {
   display: flex;
   justify-content: space-between;
 }
+
```

Comment:
- [ ] In your Readme, I cannot find instructions for testing your app.
- [ ] In your Readme, you don't have proper instructions step by step to run your application.